src/main/java/net/rptools/maptool/model/player/PlayerZoneListener.java

```diff
@@ -54,6 +59,8 @@ public void OnZoneLoading(ZoneLoading event) {
 
   @Subscribe
   public void OnZoneLoaded(ZoneLoaded event) {
```

Comment:
You shouldn't add your code to this class.
Instead you create a new class that listens for the event and register it with the event bus the same way that this class has been registered.

Reply:
I will attempt to do so.

Reply:
This should be correct (and moved the `onChangeMap` to the legacy switch case)